src/scheduler/node_info.c

```diff
@@ -3613,8 +3613,9 @@ eval_simple_selspec(status *policy, chunk *chk, node_info **pninfo_arr,
 					for (k = 0; ninfo_arr[k] != NULL; k++) {
 						if (ninfo_arr[k]->nodesig_ind == ninfo_arr[i]->nodesig_ind) {
 							ninfo_arr[k]->nscr.visited = 1;
-							schdlogerr(PBSEVENT_DEBUG3, PBS_EVENTCLASS_NODE, LOG_DEBUG,
-								ninfo_arr[k]->name, NULL, err);
+							if (i != k)
```

Comment:
I just had a thought.  This will print messages out of sorted order.  Maybe we want to write a message between the check_available_resources() and the for loop saying something like, "Marking nodes with signature %s ineligible", ninfo_arr[i]->nodesig